packages/unmock-core/src/service/service.ts

```diff
@@ -55,58 +66,165 @@ export class Service implements IService {
     return this.matcher.matchToOperationObject(sreq);
   }
 
-  public verifyRequest(method: HTTPMethod, endpoint: string) {
-    // Throws if method + endpoint are invalid for this service
+  public updateState({
+    method,
+    endpoint,
+    // @ts-ignore TODO
+    newState,
+  }: IStateInput): { success: boolean; error?: string } {
+    // Four possible cases:
```

Comment:
I think all such detailed documentation should rather be part of the JSDoc documentation for the method and not "inline" in the code?

Reply:
They're remove in a future PR, these were internal notes :)